deploy/templates/components/csv/clusterserviceversion.yaml

```diff
@@ -97,4 +97,4 @@ spec:
     url: https://devfile.io
   # uncomment the below when devworkspace-operator.v0.8.0 is availble
   # replaces: devworkspace-operator.v0.8.0
-  version: 0.9.0
+  version: 0.10.0
```

Comment:
I think we can uncomment the replaces now, we will also have to do that in the 0.9.x branch as well

Reply:
well, 0.9.0 is not available yet... Should `replaces: devworkspace-operator.v0.8.0` be there for time being?

Reply:
Yeah, that makes sense